src/ekomark/data/operators.py

```diff
@@ -17,7 +17,7 @@
             n_integration_cores=0,
             debug_skip_non_singlet=False,
             debug_skip_singlet=False,
-            Q2grid=[100],
+            mugrid=[100],
```

Comment:
```suggestion
            mugrid=[10],
```

Reply:
This makes sense, it is a consistent update :)

I'll add it.